driver/objgen/objgen.go

```diff
@@ -44,7 +53,7 @@ func Get(vars map[string]string) (*Output, error) {
 			return nil, fmt.Errorf("can't parse variable list from template: %w", err)
 		}
 
-		for key, _ := range crdVars {
+		for key := range crdVars {
```

Comment:
`crdVars` is the leftover name (initially I put it with CRD template), better to use just vars if not conflicts.

Reply:
Thanks! I'll rename that.